packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -120,6 +120,7 @@ export class RatingBase extends BaseComponent<IRatingProps, IRatingState> {
             key={ i }
             { ...((i === Math.ceil(this.state.rating as number)) ? { 'data-is-current': true } : {}) }
             onFocus={ this._onFocus.bind(this, i) }
+            onClick={ this._onFocus.bind(this, i) } // For Safari & Firefox on OSX
```

Comment:
This means that when yo click, it will call _onFocus 2 times. Could you also adjust _onFocus to only do work if the value is different?

Reply:
Added conditional in the _onFocus function